rye/src/cli/shell.rs

```diff
@@ -36,14 +76,29 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
         .context("failed to sync ahead of shell")?;
 
     let venv_path = pyproject.venv_path();
-    let venv_bin = venv_path.join("bin");
+    let venv_bin = if env::consts::OS == "windows" {
```

Comment:
I think it should be determined in runtime instead of compile time, so using `env::consts::OS == "windows"` instead of `cfg!(windows)`.

Maybe we will using binaries build in windows and runs on linux (wine?).

Reply:
Note that this conditional - bin vs Scripts - is also done once already in rye/src/consts.rs

Reply:
Updated, thanks for the review!